lib/dockerize_rails/constants.rb

```diff
@@ -17,40 +17,48 @@ module Constants
     DATABASE_HOST_REMOTE = 'remote'.freeze
 
     COMMANDS = {
-      configure: {
+        configure: {
         aliases: %I[configure c rc cr],
         help: "Generates '#{DOCKERIZE_RAILS_CONFIG_FILE_NAME}'".freeze,
         params: {
           :'--tiny' => 'generates shorter config file, skipping all descriptions'.freeze
         }
       },
-      dockerize: {
+        dockerize: {
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.

<!-- Comment by [SideCI](https://sideci.com) -->